src/Compilers/CSharp/Portable/Symbols/Symbol_Attributes.cs

```diff
@@ -405,7 +409,7 @@ internal virtual void PostDecodeWellKnownAttributes(ImmutableArray<CSharpAttribu
                         {
                             var boundAttribute = boundAttributeArray[i];
                             Debug.Assert(boundAttribute is not null);
-                            NullableWalker.AnalyzeIfNeeded(binders[i], boundAttribute, boundAttribute.Syntax, diagnostics.DiagnosticBag);
+                            NullableWalker.AnalyzeIfNeeded(binders[i], boundAttribute, boundAttribute.Syntax, diagnostics.DiagnosticBag!);
```

Comment:
Why is the suppression necessary? We have an assertion already at the top of the method: `Debug.Assert(diagnostics.DiagnosticBag is not null);`